src/app-layout/visual-refresh-toolbar/multi-layout.ts

```diff
@@ -69,15 +69,18 @@ function mergeProps(ownProps: SharedProps, additionalProps: ReadonlyArray<Partia
   return Object.keys(toolbar).filter(key => key !== 'ariaLabels').length > 0 ? toolbar : null;
 }
 
-export function useMultiAppLayout(props: SharedProps) {
+export function useMultiAppLayout(props: SharedProps, isEnabled: boolean) {
   const [registration, setRegistration] = useState<RegistrationState<SharedProps> | null>(null);
   const { forceDeduplicationType } = props;
 
   useLayoutEffect(() => {
+    if (!isEnabled) {
```

Comment:
Yes, not covered in unit tests (because there is no intersection observer API in JSDOM). But there are integration tests, so we are safe